aider/main.py

```diff
@@ -562,6 +568,18 @@ def scrub_sensitive_info(text):
     if args.message:
         io.tool_output()
         coder.run(with_message=args.message)
+    elif args.message_file:
+        try:
+            with open(args.message_file, 'r', encoding='utf-8') as file:
```

Comment:
This should use io.read_text() so that it can honor the user's `--encoding` setting.

Reply:
Done!
Thank you for learning via code review! Now I learned that `io.read_text()` benefit is honoring `--encoding` !